sdks/java/core/src/test/java/org/apache/beam/sdk/io/FileSystemsTest.java

```diff
@@ -68,6 +68,11 @@ public void testGetLocalFileSystem() throws Exception {
     assertTrue(
         FileSystems.getFileSystemInternal(toLocalResourceId("File://home").getScheme())
             instanceof LocalFileSystem);
+    if (SystemUtils.IS_OS_WINDOWS) {
+      assertTrue(
+          FileSystems.getFileSystemInternal(toLocalResourceId("c:\\home\\").getScheme())
+              instanceof LocalFileSystem);
+    }
```

Comment:
Can you also add a test for "No filesystem found for scheme "?

Reply:
thanks @jkff . Added the test cases